src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -781,7 +787,8 @@ internal interface ISpecializedEventDefinition : IEventDefinition
         /// type of a generic type instance), then the unspecialized member refers to a member from the unspecialized containing type. (I.e. the unspecialized member always
         /// corresponds to a definition that is not obtained via specialization.)
         /// </summary>
-        IEventDefinition/*!*/ UnspecializedVersion
+        [NotNull]
```

Comment:
Not sure I understand why we need a `NotNull` attribute here?

Reply:
➡️ It's not necessary, but it codifies the original intent as opposed to leaving a question of whether it just hasn't been converted to nullable yet. This form forces someone to reconsider a decision to define the property as nullable if some implementation later on appears to necessitate it.

Reply:
This isn't ambiguous. The file has been converted to nullable, and it's not marked with a ?. Consider reverting, and the other attributes that were added that are unnecessary.

Reply:
> The file has been converted to nullable...

This is not necessarily correct. Since the usages of these interfaces have not all been converted, I cannot be certain that the changes made in the conversion match reality. During follow-up work, I did find a couple inconsistencies that needed to be addressed. I'm leaving the attributes in place for now in the event they are helpful to someone else doing other follow-up work.